docs/src/pages/landing-page/Index.vue

```diff
@@ -134,13 +134,55 @@
           <div class="lp-heading lp-heading-large">
             What our community thinks of quasar
           </div>
-          <div class="text-size-20 text-lp-light text-weight-medium">
+          <div class="lp-heading lp-heading-small">
             "You've never heard of Quasar? It's the framework that made a mobile
-            app, desktop app and browser extension altogether in less than
-            12minutes."
+            app, desktop app and browser extension altogether in less than 12 minutes."
           </div>
         </div>
       </div>
+      <q-carousel
+        v-model="slide"
+        transition-prev="scale"
+        transition-next="scale"
+        swipeable
+        animated
+        navigation
+        :autoplay="true"
+        padding
+        arrows
+        height="400px"
+        class="bg-transparent rounded-borders"
+      >
+        <template v-slot:navigation-icon="{ active, btnProps, onClick }">
+          <q-btn
+            v-if="active"
+            size="xs"
+            :icon="btnProps.icon"
+            color="lp-primary"
+            flat
+            round
+            dense
+            @click="onClick"
+          />
+          <q-btn
+            v-else
+            size="6px"
+            :icon="btnProps.icon"
+            color="grey-8"
+            flat
+            round
+            dense
+            @click="onClick"
+          />
+        </template>
+        <q-carousel-slide :name="i" class="showcase-cards" v-for="(showcaseContent, i) in showcaseContents" :key="i">
+          <div class="my-grid">
+            <div style="width: 90%" v-for="(showcases, i) in showcaseContent" :key="i">
+              <twitter-card :card-content="showcases"/>
```

Comment:
The `card-content` seems always the same except for the link, as it seems just a fallback
Can you explain how is this meant to work?

Reply:
I have an array, whose elements are arrays of tweets (3 tweets per array). So I go through the larger array and display the 3 tweets per carousel slide.  So the content of card-content is not same through out. Once this is done, I'll talk with Nick to get all the real tweets that will go in there.

Reply:
Ok, but the common structure seems the same, is there a way to keep the structure as normal HTML and only pass-by the twitter id/link/whatever identifier you're using to load it later on?